cognee/infrastructure/files/storage/S3FileStorage.py

```
@@ -1,24 +1,33 @@
 
     storage_path: str
-    s3: s3fs.S3FileSystem
+    s3: "s3fs.S3FileSystem"
 
     def __init__(self, storage_path: str):
+        try:
+            import s3fs
+        except ImportError:
+            raise ImportError(
+                's3fs is required for S3FileStorage. Install it with: pip install cognee"[aws]"'
+            )
```

Ruff (0.13.1)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)